src/tests/stdlib/memcpy.c

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file
+ *
+ * @date Mar 22, 2020
+ * @author: Athreya Nagaraj
+ */
+
+#include <string.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("memcpy test suit");
+
+TEST_CASE("memcpy with zero size") {
+	char src[] = "string";
+	char dst[sizeof(src) + 1];
+
+	test_assert_equal(strlen(src), strlen(memcpy(dst, src, 0)));
```

Comment:
What do you test here?
Have you checked how your tests work?

Reply:
I wanted to check for the test case where memcpy with zero as parameter for number of bytes does not copy anything. I've realized that `test_assert_equal(strlen(src), strlen(memcpy(dst, src, 0)));` will not work. Now I've changed it so that dst is a zero-length string and has to remain so after the call to memcpy. If it changes then the test will fail. 

> Have you checked how your tests work?

Yes I've changed the `templates/x86/qemu/mods.config` file to run the test suite of memcpy. I have not added it to the PR though.